README.md

```diff
@@ -94,7 +95,13 @@ We can search DebridMediaManager hash lists which are hosted on GitHub. This all
    ```
    GithubSettings__PAT=<YOUR TOKEN HERE>
    ```
+### Configure external access
 
+What you will need:
+1. Domain or subdomain that points toward your ip. You can use [DuckDNS](duckdns.org) for a free subdomain. [Installation instructions](http://www.duckdns.org/install.jsp) are provided to keep your IP updated.
+2. Ports 80 and 443 opened on your router/gateway adn forwarded to your KNightcrawler server. Refer to [PortForward.com](https://portforward.com/).
+
```

Comment:
I'd like to add here a note about the security risks that comes with opening there ports on your router and to not do this blindly. For external access, I'd specify that it's better to either use a VPN, or have at least some IP restrictions. This app was coded by a lot of hobbyist and was not built to be publicly hosted.

Reply:
> I'd like to add here a note about the security risks that comes with opening there ports on your router and to not do this blindly. For external access, I'd specify that it's better to either use a VPN, or have at least some IP restrictions. This app was coded by a lot of hobbyist and was not built to be publicly hosted.

Caddy, working as a reverse proxy, should handle any vulnerabilities with Knightcrawler since it operates as a proxy. From my understanding of how ports work, vulnerabilities are only possible for the service exposed through that port, which in this case is Caddy. Personally, I trust Caddy from a security standpoint. I haven't brushed up on networking in a long time, so I may be incorrect.

In my opinion, a VPN is inconvenient for the average user. I also haven't been able to get any add-ons to work completely locally due to them only being http. Caddy automatically sets up https with absolutely no user input.

Reply:
Good point. That's enough for me.